eclipse-theia/theia
# Upgrade express to 4.21.0

#### What it does

Fixes #14258, #14259

Contributed on behalf of STMicroelectronics

#### How to test
Make sure electron & browser versions still start up and work O.K. No particular feature to test.

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
